code/tests/search_utilities/test_azure_search_handler.py

```diff
@@ -363,3 +372,33 @@ def test_semantic_search_with_advanced_image_processing(
         query_answer="extractive",
         top=handler.env_helper.AZURE_SEARCH_TOP_K,
     )
+
+
+def test_delete_from_index(handler, mock_search_client, mock_blob_client):
+    # given
+    blob_url = "https://example.com/blob"
+    filter_value = f"source eq '{blob_url}_SAS_TOKEN_PLACEHOLDER_'"
+    documents = [
+        {"id": 1, "title": "file1"},
+        {"id": 2, "title": "file2"},
+        {"id": 3, "title": "file1"},
+        {"id": 4, "title": "file3"},
+    ]
+    handler.search_client.search.return_value = documents
+    files_to_delete = {
+        "file1": [1, 3],
+        "file2": [2],
+        "file3": [4],
+    }
+    ids_to_delete = []
+    for filename, ids in files_to_delete.items():
```

Comment:
I always try to keep logic in tests to a minimum. It can sometimes lead to bugs. I know it's duplication, but what do you think about changing this to a hardcoded list `ids_to_delete = [1,2,3,4]` ?

Reply:
Updated